parallaxinc/solo
# Fix CSS errors in graphing download

Addresses #138 
Uses substantially less CSS to generate a downloadable SVG image snapshot from a graphing window.

To test:

- Create a program that graphs.  
- Click the camera icon after the graph has filled with some data.
- Open the downloaded graph using a browser or image viewer.
- Verify that the graph appears it you expect (labels are present and aligned well, grid is visible, etc.)
- Test again in xy mode.
